package.json

```diff
@@ -59,7 +59,7 @@
     "nodes:reset": "run-s nodes:clean nodes:up",
     "packages:version": "node ./scripts/version.js",
     "prettier:check": "prettier --check . !./packages/docs",
-    "prettier:format": "prettier --write .",
+    "prettier:format": "prettier --write . && pnpm -r --filter=docs exec prettier --write .",
```

Comment:
why we have to change this configuration here?

Reply:
otherwise prettier uses a later version on the docs that causes the build to fail